idl/chromadb/proto/coordinator.proto

```diff
@@ -28,6 +28,18 @@ message GetDatabaseResponse {
   reserved "status";
 }
 
+message ListDatabasesRequest {
+  string tenant = 1;
+  optional int32 limit = 2;
+  optional int32 offset = 3;
+}
+
+message ListDatabasesResponse {
+  repeated Database databases = 1;
+  reserved 2;
```

Comment:
field number 2 was reserved for other message types because I removed the "status" field that had a number 2. I think we don't need to reserve it for this message type.